lib/msal-common/src/response/ResponseHandler.ts

```diff
@@ -299,7 +299,7 @@ export class ResponseHandler {
         resourceRequestUri?: string): Promise<AuthenticationResult> {
         let accessToken: string = "";
         let responseScopes: Array<string> = [];
-        let expiresOn: Date | null = null;
+        let expiresOn: Date;
```

Comment:
This implicitly becomes `Date | undefined`, what was the original issue?

Reply:
Below we assign the value in the AuthenticationResult object to this variable, which caused a type mismatch.